src/main/java/com/dumbdogdiner/stickyapi/common/util/Debugger.java

```diff
@@ -71,8 +71,26 @@ public Debugger(Class<?> clazz) {
      */
     public void print(Object object, Object... args) {
         if (enabled) {
-            logger.info(String.format(COLOR + "[" + ++logCount + " | " + clazz.getSimpleName()
-            + ".class: "+Fi5vGG6kBJbVhpjH3p8PaubeS2Mdtps()+"] \u00A7r" + object + " | " + ((System.nanoTime() - startTime) / 1e3) + "\u00B5", args));
+            int result = -1;
+            boolean thisOne = false;
+            int thisOneCountDown = 1;
+            StackTraceElement[] elements = Thread.currentThread().getStackTrace();
+            for(StackTraceElement element : elements) {
+                String methodName = element.getMethodName();
+                String methodClassName = element.getClassName();
+                int lineNum = element.getLineNumber();
+                if(thisOne && (thisOneCountDown == 0)) {
+                    result = lineNum;
+                    break;
+                } else if(thisOne) {
+                    thisOneCountDown--;
+                }
+                if(methodName.equals("print") && methodClassName.equals(getClass().getName())) {
```

Comment:
This has a high potential of breaking... 
If another plugin has a lib or it's own Debugger.java class, and it happens to have print, it will get that method instead of this one. Please beware of this, that is the very reason I have it be extra unique.

Reply:
I could change it to be package and class based. Let me try to do that.